pages/instructorAssessmentInstances/instructorAssessmentInstances.ejs

```diff
@@ -93,10 +100,86 @@
         <div class="card-header bg-primary text-white d-flex align-items-center">
           <%= assessment_set.name %> <%= assessment.number %>: Assessment instances
           <div class="ml-auto">
-            <button type="button" class="btn btn-sm btn-light" tabindex="0"
-               data-toggle="modal" data-target="#deleteAllAssessmentInstancesModal">
-              <i class="fa fa-times" aria-hidden="true"></i> Delete all assessment instances
-            </button>
+            <div class="dropdown d-flex flex-row">
+              <button type="button" class="btn btn-light dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                Action for all instances <span class="caret"></span>
+              </button>
+              <div id="PopoverSetTimeLimitAll"
+                   tabindex="0" data-toggle="popover" data-trigger="manual" data-container="body"
+                   data-html="true" data-placement="auto" title="Set Time Limit"
+                   data-content="
+                                 <p>
+                                   Time limit: <%= time_limit_list %><br/>
+                                   Remaining time: <%= remaining_time_range %>
```

Comment:
I found these fields a bit confusing (it was only after reading the code that I understood them). Maybe they aren't really needed? I feel like if I'm proctoring an exam and want to add 5 min to everyone, I already have a sense of what's going on.

Reply:
I think they are useful when setting a fixed time instead of adding/subtracting. What if I use "Total time allotted" instead of "Time limit"?

Reply:
I changed it to Total time limit. I'd prefer to keep the list of possible time limits, as that would remind proctors if there is a list of students with different total time, and that they need to consider those when computing the remaining time.